app/src/main/java/org/greenstand/android/TreeTracker/view/TextButton.kt

```diff
@@ -121,6 +121,25 @@ fun ApprovalButton(
     }
 }
 
+@Composable
+fun InfoButton(modifier: Modifier = Modifier, isEnabled: Boolean,
+               onClick: () -> Unit,shape: DepthSurfaceShape = DepthSurfaceShape.Circle) {
```

Comment:
Please make sure to use proper spacing throughout the code. You can refer to the rest of the code to see how we use spacing as well as the placement of curly braces. Android studio should auto format code for you with a certain button.

Reply:
Thanks for highlight.! Addressed.